dotCMS/src/integration-test/java/com/dotcms/mock/request/CachedParameterDecoratorTest.java

```diff
@@ -0,0 +1,39 @@
+package com.dotcms.mock.request;
+
+import com.dotcms.IntegrationTestBase;
+import com.dotcms.datagen.LanguageDataGen;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.portlets.languagesmanager.model.Language;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class CachedParameterDecoratorTest extends IntegrationTestBase {
+
+
+    @BeforeClass
+    public static void prepare() throws Exception{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4617651)